src/components/footer.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component } from '@angular/core';
+
+import AngularBolivia from '@assets/images/angular-bolivia.webp';
+import AngularWhite from '@assets/images/angular-white-logo.webp';
+import XLogo from '@assets/images/x-logo.svg';
+import LinkedinLogo from '@assets/images/linkedin-logo.svg';
+import GithubLogo from '@assets/images/github-logo.svg';
+import EmailLogo from '@assets/images/email-logo.svg';
+
+@Component({
+  selector: 'app-footer',
+  standalone: true,
+  template: `
+    <footer class="bg-[#111828] text-white py-12">
+      <div class="container mx-auto px-4">
+        <div class="flex flex-col md:flex-row md:justify-between gap-12">
+          <div class="flex flex-col md:items-start gap-4">
+            <div class="flex flex-row items-center gap-4">
+              <img [src]="ANGULAR_WHITE" alt="Logo de Angular" />
+              <h2 class="text-[#FFFFFF] text-2xl font-bold">{{
+                EVENT_TITLE
+              }}</h2>
+            </div>
+            <p class="text-left md:text-left max-w-md text-[#8E9196]">
+              Empoderamos a la próxima generación de innovadores tecnológicos a
+              través de tutoría, recursos y desarrollo de comunidades.
+            </p>
+          </div>
+          <div class="flex flex-col items-center gap-2">
+            <p class="text-xl" style="font-family: 'Inter Tight', sans-serif;"
```

Comment:
Not sure if we really want to import `Inter Tight`, it's only 50kb, but it's use only here